h/accounts/test/views_test.py

```diff
@@ -339,37 +347,33 @@ def test_forgot_password_creates_no_activations_when_validation_fails(activation
 
 
 @forgot_password_fixtures
-def test_forgot_password_creates_activation_for_user(activation_model,
-                                                     authn_policy):
+def test_forgot_password_create_no_activation_for_user(activation_model,
```

Comment:
Just delete this test. We're not going to check that every component that's not supposed to be used by `forgot_password` isn't used by `forgot_password` :)